x-pack/plugins/cases/public/components/all_cases/index.tsx

```diff
@@ -157,144 +123,28 @@ export const AllCases = React.memo<AllCasesProps>(
       setSelectedCases,
     } = useGetCases();
```

Comment:
To me it seems like we should keep the table logic all here because this is where the data is manipulated by filters. If we broke the table and the filter controls into separate components, we'd either have to use context or redux to sync the state between the new components. This seems simpler.